plugins/typo_squatter/komand_typo_squatter/triggers/search_certstream/trigger.py

```diff
@@ -1,56 +1,51 @@
-import komand
-import time
-import json
+import insightconnect_plugin_runtime
 import certstream
 import re
 import Levenshtein
 from komand_typo_squatter.util import utils
-from .schema import SearchCertstreamInput, SearchCertstreamOutput
+from .schema import SearchCertstreamInput, SearchCertstreamOutput, Input, Output, Component
 
 
-class SearchCertstream(komand.Trigger):
+class SearchCertstream(insightconnect_plugin_runtime.Trigger):
     def __init__(self):
         super(self.__class__, self).__init__(
             name="search_certstream",
-            description="Searches certstream for new certs matching query",
+            description=Component.DESCRIPTION,
             input=SearchCertstreamInput(),
             output=SearchCertstreamOutput(),
         )
+        self.domain = ""
+        self.query = ""
+        self.levenshtein = 0
 
-    def callback(self, message, context):
+    def callback(self, message, context):  # pylint: disable=unused-argument
         """Callback handler for certstream events."""
-        if message["message_type"] == "heartbeat":
+        message_type = message.get("message_type")
```

Comment:
Can we put a default case here for message types?
Incase more message_types are added in the future

Reply:
Sure, I can throw an exception at the end if a type other than these two is returned. Currently, only these two types of messages are returned.